### ISSUE TITLE
Revs: vertically scrolling palette errors

### ISSUE THREAD

**Issue by SteveFosdick** [stardot/b-em#83]
Upon loading Acornsoft Revs the first graphics screen (not mode 7) has a vertical scrolling effect where the palette is incorrect this leading to a pretty ugly screen.  This is demonstrated by these screen dumps:
![revs1](https://user-images.githubusercontent.com/1048671/57961368-67d09b80-7906-11e9-9401-9635d083e54d.png)
![revs2](https://user-images.githubusercontent.com/1048671/57961372-6acb8c00-7906-11e9-8335-2303a4ae8f3e.png)
![revs3](https://user-images.githubusercontent.com/1048671/57961374-6e5f1300-7906-11e9-9dbc-bb067c2a34b6.png)
![revs4](https://user-images.githubusercontent.com/1048671/57961375-728b3080-7906-11e9-896a-07eecb63799c.png)
![revs5](https://user-images.githubusercontent.com/1048671/57961379-761eb780-7906-11e9-8e19-29af64e81c70.png)

**Comment by SteveFosdick** [comment]
Some discussion at https://stardot.org.uk/forums/viewtopic.php?f=4&t=17147.  Fixed by re-instating the first of these two lines:
```
if (frameodd) interline = (crtc[8] & 1);
interlline = frameodd && (crtc[8] & 1);
```
where I had removed the first one as, not spotting the subtly different variables names, I had assume it was redundant as the result would be overwritten by the 2nd line.  This had the effect of making the longer field in each interfaced frame shorter thus making the frame shorter overall and meaning the position of the palette change moved gradually down the screen.
